python-sdk/src/langwatch/experiment/experiment.py

```
@@ -57,6 +60,17 @@
 _tracer = trace.get_tracer(__name__)
 
 
+def _is_dataframe(obj: Any) -> bool:
```

Ruff (0.15.21)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `obj`

(ANN401)